tests/lib/outputs/html/html_test.py

```diff
@@ -0,0 +1,370 @@
+                        },
+                        // Show all filters
+                        targets: [0, 1, 2, 3, 5, 7]
+                    }
+                ]
+            });
+            var maxLength = 30;
+            // ReadMore ReadLess
+            $(".show-read-more").each(function () {
+                var myStr = $(this).text();
+                if ($.trim(myStr).length > maxLength) {
+                    var newStr = myStr.substring(0, maxLength);
+                    var removedStr = myStr.substring(maxLength, $.trim(myStr).length);
+                    $(this).empty().html(newStr);
+                    $(this).append(' <a href="javascript:void(0);" class="read-more">read more...</a>');
+                    $(this).append('<span class="more-text">' + removedStr + '</span>');
+                }
+            });
+            $(".read-more").click(function () {
+                $(this).siblings(".more-text").contents().unwrap();
+                $(this).remove();
+            });
+        });
+    </script>
+</body>
+
+</html>
+"""
+
+
+@pytest.fixture
+def generate_finding():
+    return Finding(
+        auth_method="OAuth",
+        timestamp=datetime.now(),
+        account_uid="12345",
+        account_name="Example Account",
+        account_email="example@example.com",
+        account_organization_uid="org-123",
+        account_organization_name="Example Org",
+        account_tags=["tag1", "tag2"],
+        finding_uid="finding-123",
+        provider="aws",
+        check_id="check-123",
+        check_title="Example Check",
+        check_type="Security",
+        status=Status("FAIL"),
+        status_extended="Extended status",
+        muted=False,
+        service_name="Example Service",
+        subservice_name="Example Subservice",
+        severity=Severity("critical"),
+        resource_type="Instance",
+        resource_uid="resource-123",
+        resource_name="Example Resource",
+        resource_details="Detailed information about the resource",
+        resource_tags="tag1,tag2",
+        partition="aws",
+        region="us-west-1",
+        description="Description of the finding",
+        risk="High",
+        related_url="http://example.com",
+        remediation_recommendation_text="Recommendation text",
+        remediation_recommendation_url="http://example.com/remediation",
+        remediation_code_nativeiac="native-iac-code",
+        remediation_code_terraform="terraform-code",
+        remediation_code_cli="cli-code",
+        remediation_code_other="other-code",
+        compliance={"compliance_key": "compliance_value"},
+        categories="category1,category2",
+        depends_on="dependency",
+        related_to="related finding",
+        notes="Notes about the finding",
+        prowler_version="1.0",
+    )
+
+
+class TestHTML:
+    def test_transform(self, generate_finding):
+        findings = [generate_finding]
+
+        # Clear the data from CSV class
```

Comment:
```suggestion
        # Clear the data from HTML class
```
Make this for all the comments please 🙏🏼

Reply:
Done